packages/block-library/src/media-text/index.js

```diff
@@ -64,7 +64,7 @@ const blockAttributes = {
 	},
 	isStackedOnMobile: {
 		type: 'boolean',
-		default: false,
+		default: true,
```

Comment:
Unless I'm mistaken, this would be considered a breaking change, and would invalidate existing Media & Text blocks.

See also: https://wordpress.org/gutenberg/handbook/designers-developers/developers/block-api/block-deprecation/